webpack/JobWizard/steps/AdvancedFields/AdvancedFields.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import { Form } from '@patternfly/react-core';
+import { selectJobTemplate } from '../../JobWizardSelectors';
+import {
+  EffectiveUserField,
+  TimeoutToKillField,
+  PasswordField,
+  KeyPassphraseField,
+  EffectiveUserPasswordField,
+  ConcurrencyLevelField,
+  TimeSpanLevelField,
+} from './Fields';
+
+export const AdvancedFields = ({ advancedValue, setAdvancedValue }) => {
+  const jobTemplate = useSelector(selectJobTemplate);
+  const effectiveUser = jobTemplate.effective_user;
```

Comment:
didn't you also pass it from the server https://github.com/theforeman/foreman_remote_execution/pull/553/files#diff-494fb439c4dcbef79b8470f33ba943aa649b2d8e698a9bd26c5353e9e33158f7R15

I guess no need to get it directly again from `jobTemplate`

Reply:
This is how I pass it from the server, 
```
jobTemplate= {
    job_template : { ... },
    effective_user : { ... }
}
```